braid/_braid.c

```diff
@@ -2535,6 +2535,39 @@ _braid_FRefineSpace(braid_Core   core,
    return _braid_error_flag;
 }
 
+/*----------------------------------------------------------------------------
+ * Helper routine for periodic problems.  Compute the max number of levels
+ * from user settings
```

Comment:
Perhaps this would be a better comment?
* Helper routine for periodic problems
* Computes the total number of times that Braid can coarsen, given npoints total time-points,
* and the min_coarse, max_levels, and cfactors values stored in core

And could function title be something like 
_braid_ComputeNumCoarsenings_Helper() ?
I feel like that may be more informative